main/acle.md

```diff
@@ -1072,6 +1073,30 @@ including the header:
 
 Including `<arm_sme.h>` also includes [`<arm_sve.h>`](#arm_sve.h).
 
+### Predefined feature macros and header files
+
+Testing a feature macro returns the compiler's availability for the feature
+via ACLE documented intrinsics and inline assembly. This property can be
+independent of where the test is performed. There should be no assumption on
+the order or context in wich the preprocessor macros are evaluated.
```

Comment:
I find the word 'test' confusing, would it make more sense to speak of 'evaluating' ?

```suggestion
Evaluating a feature macro returns the availability of intrinsics and inline
assembly for that feature, but no assumptions should be made on the order or
context in which the preprocessor macros are evaluated.
```